src/app/learn/get-started/prereqs/ssl-certificate/index.html

```diff
@@ -11,13 +11,24 @@
   </stache-page-summary>
 
   <stache-page-anchor>
-    Installation for Mac
+    Automated installation for Linux, Mac, and Windows
+  </stache-page-anchor>
+
+  <sky-code-block>
+    skyux certs trust
+  </sky-code-block>
+
+  <p>To learn about other options, see the <a routerLink="/learn/reference/cli-commands/certs"><sky-code>skyux certs</sky-code> documentation</a>.</p>
+
+  <stache-page-anchor>
+    Manual installation for Mac
   </stache-page-anchor>
 
   <ol>
     <li>
-      <p>Download the <a href="https://raw.githubusercontent.com/blackbaud/skyux-builder/master/ssl/skyux-ca.crt">skyux-ca.crt</a>. To download, right-click the link and save it to your computer.
-      </p>
+      <sky-code-block>
```

Comment:
This looks clunky. It seems like it should just be a step that says "Run `skyux certs generate`." using a code snippet instead of a code block. Any objection to that change?

Reply:
I like it, but it'd be `skyux certs trust`.